screens/shoppingList

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image } from 'react-native';
+import FontLoad from './components/FontLoad';
+
+export default class ShoppingList extends React.Component {
+
+
+  render() {
+    return (
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+           <View style={styles.rectangleOrange} />
```

Comment:
The `<Text>` components need to be inside `<View>` components. `<Text>` components don't sit on top of `<View>` components. Whenever you see the text inside a container, you have to actually program the text to be inside the container.

Don't use absolute positioning

Think about this: This will be a scroll view, the text has to move when the user scrolls, absolute the positioning is fixed and will not move dynamically with the user.